sklearn/feature_selection/tests/test_rfe.py

```diff
@@ -30,7 +30,7 @@ def __init__(self, foo_param=0):
 
     def fit(self, X, Y):
         assert_true(len(X) == len(Y))
-        self.coef_ = np.ones(X.shape[1], dtype=np.float64)
+        self.coef_ = np.ones((1, X.shape[1]), dtype=np.float64)
```

Comment:
I once added an assertion for RFECV to test whether the coef_ has the same length on the second dimension as X has. And I think it would be better to change the coef_ into 2 dimensions even in the test case.